src/main/java/com/melon/portfoliomanager/models/User.java

```diff
@@ -38,6 +38,14 @@ public User(String username, String email, String firstName, String lastName) {
         this.lastName = lastName;
     }
 
+    public User(Long id, String username, String email, String firstName, String lastName) {
```

Comment:
Why do you need this constructor ? Do you use it only inside tests ?

Reply:
fixed